test/helpers/stub-cmr.ts

```diff
@@ -4,16 +4,25 @@ import * as cmr from 'util/cmr';
 
 type CmrMethodName = 'cmrSearchBase' | 'fetchPost' | 'cmrPostSearchBase' | 'getCollectionsByIds' | 'getVariablesByIds' | 'getVariablesForCollection' | 'queryGranulesForCollection' | 'belongsToGroup' | 'cmrApiConfig';
 
+function stubCmr(functionName: CmrMethodName, response: object): void;
```

Comment:
I think these declaration might now be necessary.

Reply:
Thanks I just removed them.